###################error########################

久しぶりに訪問♪



ここも女子率高し

バーガー袋に入れても潰れない、大口必須のボリュームバーガーなのに、女子多い

今日は、敢えて気になっていたフィッシュバーガーにポテトMにし、コーラは+100円でlarge　sizeに♪

ここは、お肉屋さんが経営する有名なグルメバーガー屋 
 Error: ;; Invalid input <+ + + 未定義語 15 その他 1 * 0 * 0 "品詞推定:名詞"
Traceback (most recent call last):
  File "/root/workspace/NLPLibrary/nlplib_pyknp2.py", line 307, in pyknp_make_commentlist
    result = kparser.parse(sentence)
  File "/root/.pyenv/versions/anaconda3-2019.07/lib/python3.7/site-packages/pyknp/knp/knp.py", line 83, in parse
    return BList(knp_lines, self.pattern, juman_format)
  File "/root/.pyenv/versions/anaconda3-2019.07/lib/python3.7/site-packages/pyknp/knp/blist.py", line 40, in __init__
    self.parse(spec)
  File "/root/.pyenv/versions/anaconda3-2019.07/lib/python3.7/site-packages/pyknp/knp/blist.py", line 106, in parse
    raise Exception("Error: %s" % string)
Exception: Error: ;; Invalid input <+ + + 未定義語 15 その他 1 * 0 * 0 "品詞推定:名詞"
###########################################
###################error########################
【目黒ランチ♫夜は焼肉屋の二毛作人気ハンバーガー店♡】


目黒のランチの課題店に訪問✧⁺⸜⸝⁺✧
食べログでも3.58と人気なので気になってました♫


駅からも3-5分位なので割と近い☆


沢山のハンバーガーのメニューに、
平日ランチだとポテトorチキンナゲットorピクルス等のあらゆるサブメニューに、
ドリンクも付いてきちゃうと嬉しいサービス✧　\\//　✧


初訪問だったので、店員さんに1番おすすめを聞くと”ハングリーヘブンバーガーチーズ”が人気とのこと 
 invalid literal for int() with base 10: ''
Traceback (most recent call last):
  File "/root/workspace/NLPLibrary/nlplib_pyknp2.py", line 307, in pyknp_make_commentlist
    result = kparser.parse(sentence)
  File "/root/.pyenv/versions/anaconda3-2019.07/lib/python3.7/site-packages/pyknp/knp/knp.py", line 83, in parse
    return BList(knp_lines, self.pattern, juman_format)
  File "/root/.pyenv/versions/anaconda3-2019.07/lib/python3.7/site-packages/pyknp/knp/blist.py", line 43, in __init__
    self._setPAS(juman_format)
  File "/root/.pyenv/versions/anaconda3-2019.07/lib/python3.7/site-packages/pyknp/knp/blist.py", line 79, in _setPAS
    tag.pas = Pas(tag.tag_id, self)
  File "/root/.pyenv/versions/anaconda3-2019.07/lib/python3.7/site-packages/pyknp/knp/pas.py", line 87, in __init__
    self.__set_args(tag_predicate.features.get("項構造"), CaseInfoFormat.PASv41)
  File "/root/.pyenv/versions/anaconda3-2019.07/lib/python3.7/site-packages/pyknp/knp/pas.py", line 186, in __set_args
    for items in self.__parse_case_analysis_items(analysis_result, case_info_format):
  File "/root/.pyenv/versions/anaconda3-2019.07/lib/python3.7/site-packages/pyknp/knp/pas.py", line 181, in __parse_case_analysis_items
    yield self.__parse_case_info_format(items, case_info_format)
  File "/root/.pyenv/versions/anaconda3-2019.07/lib/python3.7/site-packages/pyknp/knp/pas.py", line 145, in __parse_case_info_format
    eid = int(items[3])
ValueError: invalid literal for int() with base 10: ''
###########################################
###################error########################

そして、濃いめのチェダーチーズでうまくまとまってる\/


ボリュームもたっぷりで美味しい時間を過ごせました٩و
 
 invalid literal for int() with base 10: 'ボリューム'
Traceback (most recent call last):
  File "/root/workspace/NLPLibrary/nlplib_pyknp2.py", line 307, in pyknp_make_commentlist
    result = kparser.parse(sentence)
  File "/root/.pyenv/versions/anaconda3-2019.07/lib/python3.7/site-packages/pyknp/knp/knp.py", line 83, in parse
    return BList(knp_lines, self.pattern, juman_format)
  File "/root/.pyenv/versions/anaconda3-2019.07/lib/python3.7/site-packages/pyknp/knp/blist.py", line 43, in __init__
    self._setPAS(juman_format)
  File "/root/.pyenv/versions/anaconda3-2019.07/lib/python3.7/site-packages/pyknp/knp/blist.py", line 79, in _setPAS
    tag.pas = Pas(tag.tag_id, self)
  File "/root/.pyenv/versions/anaconda3-2019.07/lib/python3.7/site-packages/pyknp/knp/pas.py", line 87, in __init__
    self.__set_args(tag_predicate.features.get("項構造"), CaseInfoFormat.PASv41)
  File "/root/.pyenv/versions/anaconda3-2019.07/lib/python3.7/site-packages/pyknp/knp/pas.py", line 186, in __set_args
    for items in self.__parse_case_analysis_items(analysis_result, case_info_format):
  File "/root/.pyenv/versions/anaconda3-2019.07/lib/python3.7/site-packages/pyknp/knp/pas.py", line 181, in __parse_case_analysis_items
    yield self.__parse_case_info_format(items, case_info_format)
  File "/root/.pyenv/versions/anaconda3-2019.07/lib/python3.7/site-packages/pyknp/knp/pas.py", line 145, in __parse_case_info_format
    eid = int(items[3])
ValueError: invalid literal for int() with base 10: 'ボリューム'
###########################################
###################error########################
昼だけの人気バーガー店@目黒


夜は焼肉屋さん


・バーガー店なのに80商品もあるみたい


・平日ランチ　メキシカンバーガー　1080円


★平日はお得だ^　^　
・メニューには　""eat　or　die""の文字


ホットなのを期待して頼んだけど
ピリ辛がちょっとだけ入ってる感じなのでそんなに辛さは感じない


次はチリビーンズかチーズかルッコラにしようかな


メニュー多くて迷う(^^;

　ホワイトデー 
 list index out of range
Traceback (most recent call last):
  File "/root/workspace/NLPLibrary/nlplib_pyknp2.py", line 307, in pyknp_make_commentlist
    result = kparser.parse(sentence)
  File "/root/.pyenv/versions/anaconda3-2019.07/lib/python3.7/site-packages/pyknp/knp/knp.py", line 83, in parse
    return BList(knp_lines, self.pattern, juman_format)
  File "/root/.pyenv/versions/anaconda3-2019.07/lib/python3.7/site-packages/pyknp/knp/blist.py", line 43, in __init__
    self._setPAS(juman_format)
  File "/root/.pyenv/versions/anaconda3-2019.07/lib/python3.7/site-packages/pyknp/knp/blist.py", line 79, in _setPAS
    tag.pas = Pas(tag.tag_id, self)
  File "/root/.pyenv/versions/anaconda3-2019.07/lib/python3.7/site-packages/pyknp/knp/pas.py", line 87, in __init__
    self.__set_args(tag_predicate.features.get("項構造"), CaseInfoFormat.PASv41)
  File "/root/.pyenv/versions/anaconda3-2019.07/lib/python3.7/site-packages/pyknp/knp/pas.py", line 186, in __set_args
    for items in self.__parse_case_analysis_items(analysis_result, case_info_format):
  File "/root/.pyenv/versions/anaconda3-2019.07/lib/python3.7/site-packages/pyknp/knp/pas.py", line 181, in __parse_case_analysis_items
    yield self.__parse_case_info_format(items, case_info_format)
  File "/root/.pyenv/versions/anaconda3-2019.07/lib/python3.7/site-packages/pyknp/knp/pas.py", line 145, in __parse_case_info_format
    eid = int(items[3])
IndexError: list index out of range
###########################################
###################error########################

また目黒に来てハンバーガー気分の時は来るかなー（￣+ー￣）ご馳走様でした 
 Error: ;; Invalid input <+ + + 未定義語 15 その他 1 * 0 * 0 "品詞推定:名詞"
Traceback (most recent call last):
  File "/root/workspace/NLPLibrary/nlplib_pyknp2.py", line 307, in pyknp_make_commentlist
    result = kparser.parse(sentence)
  File "/root/.pyenv/versions/anaconda3-2019.07/lib/python3.7/site-packages/pyknp/knp/knp.py", line 83, in parse
    return BList(knp_lines, self.pattern, juman_format)
  File "/root/.pyenv/versions/anaconda3-2019.07/lib/python3.7/site-packages/pyknp/knp/blist.py", line 40, in __init__
    self.parse(spec)
  File "/root/.pyenv/versions/anaconda3-2019.07/lib/python3.7/site-packages/pyknp/knp/blist.py", line 106, in parse
    raise Exception("Error: %s" % string)
Exception: Error: ;; Invalid input <+ + + 未定義語 15 その他 1 * 0 * 0 "品詞推定:名詞"
###########################################
